src/Mutagen.ts

```diff
@@ -59,6 +59,14 @@ export class Mutagen {
         }
     }
     stop(mutagenConfigFile: string) {
+        try {
+            // Check if the mutagen project is running
+            process.execSync(`mutagen project list -f ${mutagenConfigFile}`);
+        }
+        catch (e) {
+            return;
+        }
```

Comment:
This could be a function - 'isRunning' which you could place as a check here. Before checking the mutagen project list command, you could also check for existance of the config file.

Reply:
I moved this check to a separate method. It returns false in the catch, true otherwise.
This is because the command returns a non-0 exit code when the project is not running, which throws an exception in process.execSync().

Reply:
Great, thanks!